translations/ru_RU.ts

```diff
@@ -1,966 +1,973 @@
     </message>
     <message>
         <source>Open Containing Folder</source>
-        <translation type="unfinished"></translation>
+        <translation>Открыть папку содержания</translation>
     </message>
     <message>
         <source>Reveal in File Manager</source>
-        <translation type="unfinished"></translation>
+        <translation>Показать в файловом менеджере</translation>
     </message>
     <message>
         <source>Copy path</source>
-        <translation type="unfinished"></translation>
+        <translation>Копировать путь</translation>
     </message>
     <message>
         <source>Open problem in browser</source>
-        <translation type="unfinished"></translation>
+        <translation>Открыть в задачу в браузере</translation>
     </message>
     <message>
         <source>Copy Problem URL</source>
-        <translation type="unfinished"></translation>
+        <translation>Копировать URL задачи</translation>
     </message>
     <message>
         <source>Set Codeforces URL</source>
-        <translation type="unfinished"></translation>
+        <translation>Установить Codeforces URL</translation>
     </message>
     <message>
         <source>Set CF URL</source>
-        <translation type="unfinished"></translation>
+        <translation>Установить CF URL</translation>
     </message>
     <message>
         <source>Enter the contest ID:</source>
-        <translation type="unfinished"></translation>
+        <translation>Ввести ID соревнования:</translation>
     </message>
     <message>
         <source>Enter the problem Code (A-Z):</source>
-        <translation type="unfinished"></translation>
+        <translation>Ввести код задачи (A-Z):</translation>
     </message>
     <message>
         <source>Set Problem URL</source>
-        <translation type="unfinished"></translation>
+        <translation>Задать URL задачи</translation>
     </message>
     <message>
         <source>Enter the new problem URL:</source>
-        <translation type="unfinished"></translation>
+        <translation>Задать URL для новой задачи:</translation>
     </message>
     <message>
         <source>EventLogger</source>
-        <translation type="unfinished"></translation>
+        <translation>Логгер cобытий</translation>
     </message>
     <message>
         <source>All logs except for current session has been deleted</source>
-        <translation type="unfinished"></translation>
+        <translation>Все логи, кроме логов текущей сессии, были удалены</translation>
     </message>
     <message>
         <source>CP Editor: An editor specially designed for competitive programming</source>
-        <translation type="unfinished"></translation>
+        <translation>CP Editor: редактор, созданный для спортивного программирования</translation>
     </message>
     <message>
         <source>Ctrl+N</source>
-        <translation type="unfinished"></translation>
+        <translation>Ctrl+N</translation>
```

Comment:
You can leave the translation empty to make it untranslated instead of copying the source.

Reply:
It's OK to leave them copied now, just a hint that you don't need to do this the next time.